code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -2,173 +2,375 @@
 	desc = "A .25 Caseless Rifle high-velocity bullet casing."
+	icon_state = "clrifle_c_hv"
+	spent_icon = "clrifle_c_hv-spent"
 	caliber = CAL_CLRIFLE
 	projectile_type = /obj/item/projectile/bullet/clrifle/hv
 	is_caseless = TRUE
 
 /obj/item/ammo_casing/clrifle/rubber
 	desc = "A .25 Caseless Rifle rubber bullet casing."
+	icon_state = "clrifle_c_r"
+	spent_icon = "clrifle_c_r-spent"
 	caliber = CAL_CLRIFLE
 	projectile_type = /obj/item/projectile/bullet/clrifle/rubber
 	is_caseless = TRUE
 
+/obj/item/ammo_casing/clrifle/s
+	desc = "An old .25 Caseless Rifle bullet casing."
+	icon_state = "clrifle_c_s"
+	spent_icon = "clrifle_c_s-spent"
+	caliber = CAL_CLRIFLE
+	projectile_type = /obj/item/projectile/bullet/clrifle/s
+
+/obj/item/ammo_casing/clrifle/s/prespawned
+	amount = 10
+
 //// .30 ////
 
 /obj/item/ammo_casing/lrifle
 	desc = "A .30 rifle bullet casing."
+	icon_state = "lrifle_c_l"
+	spent_icon = "lrifle_c_l-spent"
 	caliber = CAL_LRIFLE
 	projectile_type = /obj/item/projectile/bullet/lrifle
+	maxamount = 10
+
+/obj/item/ammo_casing/lrifle/Initialize()
+	. = ..()
+	var/matrix/rotation_matrix = matrix()
+	rotation_matrix.Turn(round(45 * rand(0, 32) / 2))
+	src.transform = rotation_matrix * 0.5
 
 /obj/item/ammo_casing/lrifle/practice
 	desc = "A .30 rifle practice bullet casing."
+	icon_state = "lrifle_c_p"
+	spent_icon = "lrifle_c_p-spent"
 	caliber = CAL_LRIFLE
 	projectile_type = /obj/item/projectile/bullet/lrifle/practice
 
 /obj/item/ammo_casing/lrifle/hv
 	desc = "A .30 Rifle high-velocity bullet casing."
+	icon_state = "lrifle_c_hv"
+	spent_icon = "lrifle_c_hv-spent"
 	caliber = CAL_LRIFLE
 	projectile_type = /obj/item/projectile/bullet/lrifle/hv
 
 /obj/item/ammo_casing/lrifle/rubber
 	desc = "A .30 Rifle rubber bullet casing."
+	icon_state = "lrifle_c_r"
+	spent_icon = "lrifle_c_r-spent"
 	caliber = CAL_LRIFLE
 	projectile_type = /obj/item/projectile/bullet/lrifle/rubber
 
+/obj/item/ammo_casing/lrifle/s
+	desc = "An old .30 rifle bullet casing."
+	icon_state = "lrifle_c_s"
+	spent_icon = "lrifle_c_s-spent"
+	caliber = CAL_LRIFLE
+	projectile_type = /obj/item/projectile/bullet/lrifle/s
+
+/obj/item/ammo_casing/lrifle/s/prespawned
+	amount = 10
+
 //// .60 ////
 
 /obj/item/ammo_casing/antim
 	name = "shell casing"
 	desc = "A .60 Anti-Material shell."
-	icon_state = "lcasing"
-	spent_icon = "lcasing-spent"
+	icon_state = "atmr"
+	spent_icon = "atmr-spent"
 	caliber = CAL_ANTIM
 	projectile_type = /obj/item/projectile/bullet/antim
 	matter = list(MATERIAL_STEEL = 2)
 	maxamount = 5
 
+/obj/item/ammo_casing/antim/Initialize()
+	. = ..()
+	var/matrix/rotation_matrix = matrix()
+	rotation_matrix.Turn(round(45 * rand(0, 32) / 2))
+	src.transform = rotation_matrix * 0.5
+
 /obj/item/ammo_casing/antim/prespawned
 	amount = 5
 
+/obj/item/ammo_casing/antim/s
+	name = "shell casing"
+	desc = "An old .60 Anti-Material shell."
+	icon_state = "atmr_s"
+	spent_icon = "atmr_s-spent"
+	projectile_type = /obj/item/projectile/bullet/antim/owo
```

Comment:
I'm sure it should use a different projectile.

Reply:
Oops, must have made a mistake when I was tired. I will fix it when I get home tonight.